backend/src/test/java/com/github/alaajouri/backend/controller/MongoUserControllerTest.java

```diff
@@ -19,18 +20,24 @@
 @SpringBootTest
 @AutoConfigureMockMvc
 class MongoUserControllerTest {
-    @Autowired
-    MockMvc mockMvc;
 
     @Autowired
     MongoUserRepository mongoUserRepository;
+    @Autowired
+    MockMvc mockMvc;
+    MongoUser User1;
+
+    @BeforeEach
+    void setUp() {
+        User1 = new MongoUser("1", "user", "password", "BASIC", "Alaa", "W", "55", 50, 50, 8, 3, 1500);
+    }
 
 
     @Test
     @DirtiesContext
     @WithMockUser(username = "user", password = "password")
     void getMe_whenAuthenticated_thenUsername() throws Exception {
-        mongoUserRepository.save(new MongoUser("1", "user", "password", "BASIC"));
+        mongoUserRepository.save(new MongoUser("1", "user", "password", "BASIC", "Alaa", "W", "55", 50, 50, 8, 3, 1500));
```

Comment:
why not using here the user1?

```suggestion
        mongoUserRepository.save(user1);
```